orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Pipeline.groovy

```diff
@@ -16,9 +16,11 @@
 
 package com.netflix.spinnaker.orca.pipeline.model
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
 import groovy.transform.CompileStatic
 
 @CompileStatic
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
what properties were landing in as a Pipeline that we need to ignore? (concerned we are deserializing an Orchestration somewhere to a Pipeline and not to a generic Execution and wondering what the ripple effect of that is)